<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoktorListe</class>
 <widget class="QDialog" name="DoktorListe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Doktorlar</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblAra">
       <property name="text">
        <string>Ara</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leAra"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="BtnDoktorOlustur">
       <property name="text">
        <string>Doktor Oluştur</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSil">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Sil</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btnYeniSifre">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Yeni Şifre</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
